### ISSUE TITLE
Bug: Can not use `@oxc-parser/wasm` without error

### ISSUE THREAD

**Issue by Saeid-Za** [oxc-project/oxc#2828]
Hello ! I've been trying to use `@oxc-parser/wasm` but there is an error.

```
Error: Cannot find package 'test/node_modules/@oxc-parser/wasm/package.json' imported from 'test/a.mjs'
```

To reproduce the error:

1. run `npm i @oxc-parser/wasm` in an empty folder
2. create a `test.mjs` file, having this content:

```js
import {parseSync} from "@oxc-parser/wasm"
console.log(parseSync)
```

3. Try to run the code.

**Comment by Boshen** [comment]
The state of wasm build is really confusing.

As stated in the README

> This package is built with wasm-pack build --release --target web for bundler (webpack / vite) consumption. Checkout [oxc-parser](https://www.npmjs.com/package/oxc-parser) for usage in node.js.

The original issue is about making the napi version work on stackbliz, pinging @Brooooooklyn who may know more.

**Comment by Boshen** [comment]
The original issue is about making it work on stackbliz, pinging @Brooooooklyn who may know more.

**Comment by Boshen** [comment]
We may need to publish more builds (wasi, node.js target) in the near future, but I'll close this for now as I don't have the bandwidth.

**Comment by Boshen** [comment]
Closed by #2916

Released v0.1.0 https://www.npmjs.com/package/@oxc-parser/wasm
